jsk_naoqi_robot/pepper.rosinstall

```diff
@@ -2,4 +2,41 @@
     local-name: jsk_robot
     uri: https://github.com/jsk-ros-pkg/jsk_robot.git
     version: master
+# fix qt> 5 (https://github.com/ros-naoqi/naoqi_dashboard/pull/3)
+- git:
+    local-name: naoqi_dashboard
+    uri: https://github.com/kochigami/naoqi_dashboard
+    version: kochigami-develop
+# wait for 0.5.16 release
+- git:
+    local-name: nao_robot
+    uri: https://github.com/ros-naoqi/nao_robot
+# wait for 0.1.11
+- git:
+    local-name: pepper_robot
+    uri: https://github.com/ros-naoqi/pepper_robot
```

Comment:
In fact, I also use kochigami-develop for `pepper_robot` package.
I'll consider whether we really need kochigami-develop or not.

Reply:
The difference is https://github.com/ros-naoqi/pepper_robot/compare/master...kochigami:kochigami-develop
```
 +   <arg name="boot_config_file_name" value="$(arg boot_config_file_name)" />
```
And this is default behavior of `naoqi_driver.launch`
https://github.com/kochigami/naoqi_driver/blob/kochigami-develop/launch/naoqi_driver.launch